### ISSUE TITLE
Event 1-2 doesnt work.

### ISSUE THREAD

**Issue by dice4321** [mrmin123/kancolle-auto#242]
#### Environment details

* kancolle-auto version (last modified date): latest
* Viewer/browser: KC3
* Operating system: windows 10

#### Issue
Is the issue consistent (can be reproduced consistently) or is it transient (only happens sometimes)? Please check one of the boxes:

- [x] Consistent
- [ ] Transient

For some reason, the pictures work, event 1-1 works, event 1-3 works but event 1-2 doesnt work. I replaced 1-2's picture with 1-3's picture and set the variable to 1-3 and it goes to the 1-2 picture, but when I input 1-2 in the variable with 1-2's picture it does the recovery after it gets to the event maps page

![image](https://cloud.githubusercontent.com/assets/20022859/20473681/21b9a606-af77-11e6-9fad-dca40bab389f.png)

#### Console log (if applicable)
```
[replace this line with ~50 lines of console log]
```

#### Relevant config snippet (if applicable)
```

[General]
Program: KC3

RecoveryMethod: KC3

BasicRecovery: True

JSTOffset: 0

SleepCycle: 10

Paranoia: 1

SleepModifier: 0


[ScheduledSleep]

Enabled: True

StartTime: 1000

SleepLength: 4


[ScheduledStop]
Enabled: False

Mode: time

Count: 5


[Expeditions]
Enabled: False

Fleet2: 2
Fleet3: 21
Fleet4: 38


[PvP]

Enabled: False
FleetComp: 1

[Combat]
FleetComp: 2
Area: E
Subarea: 1-2
CombinedFleet: False
Nodes: 3
NodeSelects: _node_E-3-K
Formations: line_ahead, line_abreast, line_abreast
NightBattles: False, False, False
RetreatLimit: 1
RepairLimit: 1
RepairTimeLimit: 0100
CheckFatigue: True
PortCheck: True
LastNodePush: False

[SubmarineSwitch]
Enabled:True

[LBAS]

Group1Nodes:
Group2Nodes:
Group3Nodes:


[Quests]
Quests: bd1, bd2, bd3, bd4, bd5, bd6, bd7, bd8, bw1, bw2, bw3, bw4, bw5, bw6, bw7, bw8, bw9, bw10, c2, c3, c4, c8, d2, d3, d4, d9, d11, e3, e4

CheckSchedule: 3
```

**Comment by mrmin123** [comment]
It might be necessary to re-generate the panel images.. For 1-2 the LBAS symbol/text actually runs into the image generated off the wiki images.

**Comment by mrmin123** [comment]
@diceman112 could you try the images in the 243_2016_fall branch?

**Comment by dice4321** [comment]
@mrmin123 all your images work, even mines, cept there's event images outside the combat folder I misplaced that the program is reading, which are the wrong ones, my bad.

**Comment by mrmin123** [comment]
Merged your PR. For the record, sikuli (to the best of my knowledge) will start looking for images in the base `kancolle-auto.sikuli` folder, then move down to the modules for images. Having images with conflicting names up in the root `kancolle-auto.sikuli` folder will overwrite what is in any subsequent modules. I put the images in their relevant module folders just for a cleaner file structure, so it's certainly something to be aware of. I *think* moving images to a non`.sikuli` folder like I did with the old event images in the `combat.sikuli` folder makes them invalid for image matching, but I could be wrong. Either way, they are further down the directory chain so whatever is in `combat.sikuli` seems to be being matched against, despite the possibility of name collision.
